docs/src/pages/components/grid/NestedGrid.tsx

```diff
@@ -1,42 +1,33 @@
 import * as React from 'react';
-import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import Paper from '@material-ui/core/Paper';
 import Grid from '@material-ui/core/Grid';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      flexGrow: 1,
-    },
-    paper: {
-      padding: theme.spacing(1),
-      textAlign: 'center',
-      color: theme.palette.text.secondary,
-    },
-  }),
-);
-
 export default function NestedGrid() {
-  const classes = useStyles();
-
   function FormRow() {
```

Comment:
This component should be defined outside of the render method.

Reply:
Resolved 🚀